src/js/tabs/configuration.js

```diff
@@ -625,7 +625,7 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
         }
 
-        if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+        if (semver.gte(CONFIG.apiVersion, "1.41.0") && semver.lt(CONFIG.apiVersion, "1.43.0")) {
```

Comment:
To me this warning should not be version dependent - we either want to warn users using a firmware version that has dropped support for 32 kHz gyro, or we don't. Since we do not actively support users newly installing versions before 4.1, showing a warning only to them seems to be moot anyway.
Plus removing the warning entirely will result in a simplification, which is welcome.

Reply:
The thought was that if somebody was upgrading from 3.5.x which had 32K to 4.0.x or 4.1.x the warning might be useful. But maybe that's trying to hard.

Reply:
This change will most likely only be released at the same time that 4.2 is, and at that point I'd expect the majority of users who are upgrading from 3.5.x to upgrade directly to the latest version, which will be 4.2.

Reply:
Ok, then the final decision is to remove the warning completely. I will do it later.